lang/sr.json

```diff
@@ -8,7 +8,7 @@
 	},
 	"dir": "ltr",
 	"LangCode": "sr",
-	"DateTime": "d.m.yyyy HH:MM",
+	"DateTime": "m/d/yyyy h:MM aa",
```

Comment:
unfortunately, I copy pasted that lines from English file, so that's the reason it stayed that way.
so, in reality, it really is day/month/year 

I don't know what aa stands for, either, but sure is there in english file.
come to think of it, now I must change Croatian for that as well.